webui/react/src/components/kit/LineChart.tsx

```diff
@@ -23,9 +23,12 @@ import css from './LineChart.module.scss';
  * @param {string} [name] - Name to display in legend and toolip instead of Series number.
  */
 
+export const TRAIN_PREFIX = /(t_|train_|training_)/;
+export const VAL_PREFIX = /(v_|val_|validation_)/;
```

Comment:
if these are here only to import into TrialDetailsOverview, I'd think you might declare them there instead

Reply:
also would prefer a ^ in the regex, i.e. `/^(t_|train_|training_)/`